client/src/components/app.js

```diff
@@ -7,7 +7,7 @@ import GoogleMap from './GoogleMap';
 import PlacesMap from './PlacesMap';
 import Nav from './Nav';
 import Logo from './Logo';
-import Pins from './Pins';
+import Pins from './pins';
```

Comment:
I would actually like to make the directory names for components capitalized, as naming convention rather than changing this import path